nixos/modules/services/misc/renovate.nix

```diff
@@ -100,6 +100,7 @@ in
       path = [
         config.systemd.package
         pkgs.git
+        pkgs.gnupg
```

Comment:
I don't think this should be included in the default configuration, thats what `runtimePackages` is for.

Reply:
I am not sure how I missed that, I guess my brain is more scrambled these days than I realize. Validating now.